test/compile/scale.test.ts

```diff
@@ -389,11 +389,51 @@ describe('Scale', function() {
   });
 
   describe('rangeMixins()', function() {
-    // FIXME
+    describe('row', function() {
+      // TODO:
+    });
+
+    describe('column', function() {
+      // TODO:
+    });
+
+    describe('x', function() {
+      // TODO: X
+    });
+
+    describe('y', function() {
```

Comment:
Okay. Maybe I'm missing the point of this fix then. I thought this pr fixes the issue that we don't have a scale when we only define an x scale because a spec only defines y.